app/controllers/members_controller.rb

```diff
@@ -1,15 +1,8 @@
 class MembersController  < ApplicationController
   before_action :authenticate_user!, PQUserFilter
-  before_action :load_service
```

Comment:
What happens if we add another route to the controller? service not loaded (but might be premature optimisation)